release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -518,12 +518,20 @@ With this release, you can install a cluster on Nutanix by using the named, prel
 
 [id="ocp-4-18-installation-and-update-nutanix-multiple-nics_{context}"]
-==== Installing a cluster on Nutanix with up to 32 subnets
-With this release, Nutanix supports more than one subnet for the Prism Element where you deployed an {product-title} cluster to. A maximum of 32 subnets for each Prism Element is supported.
+==== Installing a cluster on Nutanix with multiple subnets
+With this release, you can install a Nutanix cluster with more than one subnet for the Prism Element into which you are deploying an {product-title} cluster. 
 
 For more information, see xref:../installing/installing_nutanix/installing-nutanix-installer-provisioned.adoc#installation-configuring-nutanix-failure-domains_installing-nutanix-installer-provisioned[Configuring failure domains] and xref:../installing/installing_nutanix/installation-config-parameters-nutanix.adoc#installation-configuration-parameters-additional-nutanix_installation-config-parameters-nutanix[Additional Nutanix configuration parameters].
 
-For an existing Nutanix cluster, you can add multiple subnets by using machine sets. For more information, see xref:../installing/installing_nutanix/nutanix-failure-domains.adoc#post-installation-configuring-nutanix-failure-domains_nutanix-failure-domains[Adding failure domains to the Infrastructure CR].
+For an existing Nutanix cluster, you can add multiple subnets by using xref:../machine_management/creating_machinesets/creating-machineset-nutanix.adoc#machineset-yaml-nutanix_creating-machineset-nutanix[compute] or xref:../machine_management/control_plane_machine_management/cpmso_provider_configurations/cpmso-config-options-nutanix.adoc#cpmso-yaml-provider-spec-nutanix_cpmso-config-options-nutanix[control plane] machine sets.
+
+[id="ocp-4-18-installation-and-update-vsphere-multiple-nics_{context}"]
+==== Installing a cluster on {vmw-full} with multiple network interface controllers (Technology Preview)
+With this release, you can install a {vmw-full} cluster with multiple network interface controllers (NICs) for a node.
+
+For more information, see xref:../installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned-network-customizations.adoc#installation-vsphere-multiple-nics_installing-vsphere-installer-provisioned-network-customizations[Configuring multiple NICs].
+
+For an existing {vmw-short} cluster, you can add multiple subnets by using xref:../machine_management/creating_machinesets/creating-machineset-vsphere.adoc#machineset-vsphere-multiple-nics_creating-machineset-vsphere[compute] or xref:../machine_management/control_plane_machine_management/cpmso_provider_configurations/cpmso-config-options-vsphere.adoc#machineset-vsphere-multiple-nics_cpmso-config-options-vsphere[control plane] machine sets.
```

Comment:
@rvanderp3 PTAL at the rel notes for vSphere

Reply:
@WenXinWei these release notes go with https://github.com/openshift/openshift-docs/pull/88235

Reply:
LGTM